packages/backend/src/shared/utils.ts

```diff
@@ -171,6 +173,53 @@ async function canApiSignatureBeProcessed(
   return true
 }
 
+export interface TenantApiSignatureResult {
+  tenant?: Tenant
```

Comment:
```suggestion
  tenant: Tenant
```
(same for `TenantedApolloContext`)